### PR TITLE
Add Notepad++ syntax colouring for Vintage BASIC.

### PR DESCRIPTION
**Pull request by markbernard**
Due to the tight nature of BASIC it can be pretty hard to parse visually in plain text. I added a syntax colouring file that can be used to view in Notepad++.

### CODE REVIEW

**Comment by coding-horror** [comment]
Did we want to add this to each folder? I guess it depends which editor people use..

**Comment by markbernard** [comment]
Maybe just leave it in one place and mention it in the documentation that it is there.
